mappings/net/minecraft/fluid/Fluid.mapping

```diff
@@ -27,10 +27,18 @@ CLASS net/minecraft/class_3611 net/minecraft/fluid/Fluid
 		ARG 1 fluid
 	METHOD method_15781 setDefaultState (Lnet/minecraft/class_3610;)V
 		ARG 1 state
-	METHOD method_15782 getVelocity (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_3610;)Lnet/minecraft/class_243;
+	METHOD method_15782 getFlowDirectionVector (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_3610;)Lnet/minecraft/class_243;
+		COMMENT Gets a 3D vector indicating the direction the given fluid state will flow towards
```

Comment:
```suggestion
		COMMENT {@return a 3D vector indicating the direction the given fluid state will flow towards}
```
The inline `@return` tag duplicates the sentence for you. https://docs.oracle.com/en/java/javase/18/docs/specs/javadoc/doc-comment-spec.html#return